public/components/report_definitions/edit/edit_report_definition.tsx

```diff
@@ -0,0 +1,62 @@
+/*
+ * Copyright 2020 Amazon.com, Inc. or its affiliates. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License").
+ * You may not use this file except in compliance with the License.
+ * A copy of the License is located at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * or in the "license" file accompanying this file. This file is distributed
+ * on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either
+ * express or implied. See the License for the specific language governing
+ * permissions and limitations under the License.
+ */
+
+import React, { useState } from 'react';
+import {
+  EuiButtonEmpty,
+  EuiFlexGroup,
+  EuiFlexItem,
+  EuiButton,
+  EuiPage,
+  EuiTitle,
+  EuiPageBody,
+  EuiSpacer,
+} from '@elastic/eui';
+import { ReportSettings } from '../report_settings';
+import { ReportDelivery } from '../delivery';
+import { ReportTrigger } from '../report_trigger';
+
+export function EditReportDefinition() {
+  return (
+    <EuiPage>
+      <EuiPageBody>
+        <EuiTitle>
+          <h1>Edit report definition</h1>
+        </EuiTitle>
+        <EuiSpacer />
+        <ReportSettings />
+        <EuiSpacer />
+        <ReportTrigger />
+        <EuiSpacer />
+        <ReportDelivery />
+        <EuiSpacer />
+        <EuiFlexGroup justifyContent="flexEnd">
+          <EuiFlexItem grow={false}>
+            <EuiButtonEmpty
+              onClick={() => {
+                window.location.assign('opendistro_kibana_reports#/');
+              }}
+            >
+              Cancel
+            </EuiButtonEmpty>
+          </EuiFlexItem>
+          <EuiFlexItem grow={false}>
+            <EuiButton fill>Save Changes</EuiButton>
```

Comment:
Does this attribute need to be assigned with any value?

Reply:
Yes, the `onClick` function for this button will be calling the APIs to update a report definition